mlrun/datastore/datastore_profile.py

```diff
@@ -500,18 +500,37 @@ class DatastoreProfilePostgreSQL(DatastoreProfile):
     port: int
     database: str = pydantic.v1.Field(
         default="postgres"
-    )  # the default maintenance database
+    )  # Default to postgres maintenance database
```

Comment:
```suggestion
    database: str = "postgres"
```
Seems like unnecessary wrapper and a comment that misinterprets the unnecessary wrapper?

Reply:
Agreed, simplified.